docs/usage-gui.md

```diff
@@ -76,16 +76,19 @@ password if it's password protected. After generation new address will be append
 ### Sending transactions
 
 As with requesting, you can click "SEND" when either wallet or account is selected. A send dialog
-will open, where you can enter the amount you want to send and receiver's address. If you click
-"SEND" on a wallet, you will have to choose accounts to use as inputs as well. Click on "select
-accounts..." text right of "FROM" and a pop-up will appear, giving you the list of the current
-wallet's accounts along with their balances. Select desired accounts by clicking on their names and
+will open, where you can choose basic or advanced version. Basic version provides single input fields
+for receiver address and send amount. Advanced version provides multiple amounts and receiver's addresses
+you want to send. You can add new receiver only after filling addresses and amounts for current receivers.
```

Comment:
probably "you want to send **to**", but I'm not sure.

Reply:
Your variant is more formal, I believe